code/game/objects/items/carp_lasso.dm

```diff
@@ -1,59 +1,79 @@
 /obj/item/mob_lasso
 	name = "space lasso"
-	desc = "Comes standard with every space-cowboy.\nCan be used to tame space carp."
+	desc = "Comes standard with every space-cowboy.\n<span class='notice'>Can be used to tame space carp.</span>"
```

Comment:
why

Reply:
Figured it looks better if the taming descriptions were colored blue, sort of like how most tool and machine descriptions with second lines do it.